### DISCUSSION TITLE
2024-06-24(월)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#266] [데일리스크럼]
아래 내용을 복사해 작성해주세요.
```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by Seoin02** [comment]
### 완료한 일
- #265
- #258

### 해야할 일
- 이슈 청산

### 어려운 일
- 이슈 넘 많아효ㅠㅠ

**Comment by seolsis** [comment]
### 완료한 일

### 해야할 일
- 결제 완료 후 렌더링
- 이슈 청산
### 어려운 일

**Comment by devwqc** [comment]
### 완료한 일
- 검색 페이지

### 해야할 일
- 이슈 청산
- 발표 자료 준비

### 어려운 일
- 파이팅팅팅!!💪
